selinux/__init__.py

```diff
@@ -71,11 +71,7 @@ def add_location(location):
 
     def get_system_sitepackages():
         """Get sitepackage locations from system python"""
-        # Do not ever use sys.executable here
-        # See https://github.com/pycontribs/selinux/issues/17 for details
-        system_python = "/usr/bin/python%s" % ".".join(
-            [str(item) for item in platform.python_version_tuple()[0:2]]
-        )
+        system_python = sys.executable
```

Comment:
This will break badly as it will pick a local python when run from within a virtualenv, and it fail to discover the system python.

Reply:
I wrote a more elaborate fix, is this one acceptable?